app/assets/javascripts/dashboard.js

```diff
@@ -1,32 +1,52 @@
-//= require leaflet.locate
-
 $(function () {
   const defaultHomeZoom = 12;
   let map;
 
   if ($("#map").length) {
-    map = L.map("map", {
-      attributionControl: false,
-      zoomControl: false
-    }).addLayer(new L.OSM.Mapnik());
-
-    const position = $("html").attr("dir") === "rtl" ? "topleft" : "topright";
-
-    L.OSM.zoom({ position }).addTo(map);
+    const position = $("html").attr("dir") === "rtl" ? "top-left" : "top-right";
 
-    L.OSM.locate({ position }).addTo(map);
+    map = new maplibregl.Map({
+      container: "map",
+      style: {
+        "version": 8,
+        "name": "OSM Raster",
+        "sources": {
+          "osm": {
+            "type": "raster",
+            "tiles": [
+              "https://tile.openstreetmap.org/{z}/{x}/{y}.png"
+            ],
+            "tileSize": 256,
+            "maxzoom": 19,
+            "attribution": "© OpenStreetMap contributors"
+          }
+        },
+        "layers": [
+          {
+            "id": "osm",
+            "type": "raster",
+            "source": "osm",
+          }
+        ],
+        "projection": { "type": "globe" },
```

Comment:
The globe is the only only thing which should be not precicely as before for users.

If osm prefers a flat eath (pun intended), this can be removed to use webmercator instead

```suggestion
```